### PR TITLE
User profil

### PR DESCRIPTION
**Pull request by jyaire**

### CODE REVIEW

File: src/Controller/UserController.php [hunk 1]
```diff
@@ -256,4 +256,48 @@ public function delete(Request $request, User $user): Response
...
+     * @return Response
+     */
+    public function showUser(User $user, CollectsRepository $collects): Response
+    {
+        if (($this->getUser()->getId()) == $user->getId()) {
+            return $this->render('user/showUser.html.twig', [
+                'user' => $user,
+            ]);
+        }
+        $this->addFlash('danger', 'Tu ne peux pas accèder au compte d\'un autre utilisateur');
```
Comment (marue59) [inline src/Controller/UserController.php]: please add Else

File: templates/infos/mentions.html.twig [hunk 2]
```diff
@@ -11,11 +11,19 @@
...
         <div class="list-group mt-2">
             <div class="flex-column align-items-start mb-4">
-                <small class="text-muted">Page en construction.</small>
+                <small class="text-muted"></small>
             </div>
             <div class="flex-column align-items-start mb-4">
-                <p class="mb-1">Titre</p>
-                <small class="text-muted">Informations.</small>
+                <p class="mb-1">Responsable de publication</p>
+                <small class="text-muted">Bipbip Mobile est édité par la société Enviro Services France.<br><br>
```
Comment (marue59) [inline templates/infos/mentions.html.twig]: please not br

File: templates/infos/protection.html.twig [hunk 3]
```diff
@@ -15,21 +15,34 @@
...
-                <small class="text-muted">Depuis ton compte client, tu peux avoir accès à l'historique de tes estimations, aux ventes réalisées, ainsi qu'aux informations personnelles que tu as fournies.<br>
-                    Depuis cette page, tu peux éventuellement modifier ces données.</bfr></small>
+                <small class="text-muted">Depuis
+                    {% if app.user.id is defined %}
+                    <a href="{{ path('user_show', { id:app.user.id}) }}">ton compte client</a>
+                    {% else %}
+                    ton compte client si tu es connecté
+                    {% endif %}
+                    , tu peux avoir accès à l'historique de tes estimations, aux ventes réalisées, ainsi qu'aux informations personnelles que tu as fournies.<br>
+                    Depuis cette page, tu peux modifier certaines données.</bfr></small>
```
Comment (marue59) [inline templates/infos/protection.html.twig]: bfr ?

File: src/Controller/UserController.php (hunk 1 above)
Comment (jyaire) [reply src/Controller/UserController.php]: else added

File: templates/infos/mentions.html.twig (hunk 2 above)
Comment (jyaire) [reply templates/infos/mentions.html.twig]: can't find other way than br with bootstrap, I will search later.

File: templates/infos/protection.html.twig (hunk 3 above)
Comment (jyaire) [reply templates/infos/protection.html.twig]: Corrected !

**Pull request merged**